NeuroConn/preprocessing/preprocessing.py

```diff
@@ -43,7 +43,7 @@ def parse_path_windows_docker(path):
         path = '/' + path[0].lower() + '/' + path[2:]
     return path
 
-def parse_fmriprep_command(data_path, fmriprep_path, fs_license_path, work_path, participant_label, skip_bids_validation, nthreads, output_spaces, fs_recon_all, task, niprep_wrapper,mem_mb, sloppy = False, system = platform.system()):
+def parse_fmriprep_command(data_path, fmriprep_path, fs_license_path, work_path, participant_label, skip_bids_validation = True, nthreads, output_spaces, fs_recon_all, task, niprep_wrapper,mem_mb, sloppy = False, system = platform.system()):
```

Comment:
Ok to skip bids validation by default. I just realized that it should be `nipreps_wrapper` instead of `niprep_wrapper`. A minor thing, but can you go over the code and make this change? just do cmd+F and then make a replacement of all instances of `niprep_wrapper`